cmd/flags.go

```diff
@@ -133,6 +134,10 @@ func FlagLogLevel(level *string) {
 	flagSet.StringVar(level, "log-level", envOrDefault(EnvLogLevel, DefaultLogLevel), "logging level")
 }
 
+func FlagProjectMetadataPath(projectMetadataPath *string) {
+	flagSet.StringVar(projectMetadataPath, "project-metadata", envOrDefault(EnvProjectMetadataPath, "./project-metadata.toml"), "path to project-metadata.toml")
```

Comment:
the default value should be grouped with the other defaults near the top of the file

Reply:
Sorry I miss the pattern here. I add a constant `DefaultProjectMetadataPath`